### ISSUE TITLE
AutoAtlas: Neural Network for 3D Unsupervised Partitioning and Representation Learning. (arXiv:2010.15987v1 [eess.IV])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#174]
AutoAtlas: Neural Network for 3D Unsupervised Partitioning and Representation Learning. (arXiv:2010.15987v1 [eess.IV])<br>
https://ift.tt/3elc0YD<br>
<p>We present a novel neural network architecture called AutoAtlas for fully unsupervised partitioning and representation learning of 3D brain Magnetic Resonance Imaging (MRI) volumes. AutoAtlas consists of two neural network components: one that performs multi-label partitioning based on local texture in the volume and a second that compresses the information contained within each partition. We train both of these components simultaneously by optimizing a loss function that is designed to promote accurate reconstruction of each partition, while encouraging spatially smooth and contiguous partitioning, and discouraging relatively small partitions. We show that the partitions adapt to the subject specific structural variations of brain tissue while consistently appearing at similar spatial locations across subjects. AutoAtlas also produces very low dimensional features that represent local texture of each partition. We demonstrate prediction of metadata associated with each subject using the derived feature representations and compare the results to prediction using features derived from FreeSurfer anatomical parcellation. Since our features are intrinsically linked to distinct partitions, we can then map values of interest, such as partition-specific feature importance scores onto the brain for visualization.</p>
<br>
<br>
via cs.CV updates on arXiv.org http://arxiv.org/<br>
